scripts/test_ws_client.py

```diff
@@ -0,0 +1,67 @@
+import asyncio
+import logging
+import socket
+
+import websockets
+
+from freqtrade.enums import RPCMessageType
+from freqtrade.rpc.api_server.ws.channel import WebSocketChannel
+
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+async def _client():
+    subscribe_topics = [RPCMessageType.WHITELIST]
```

Comment:
I've tried to mess with this script to modify this line to `subscribe_topics = [RPCMessageType.WHITELIST, RPCMessageType.ENTRY, RPCMessageType.EXIT]`.

i don't seem to be getting BUY/SELL messages though (telegram is sending them properly).

Whitelist works fine - until the first entry/exit message appears - which makes whitelist messages stop in the client (They're still sent in the server though).
Pings still continue as expected - so the connection is clearly still thre.

Also a reconnect won't "retrigger" whitelist messages - need to restart the server for this - so something in the server silently crashes / stops when it gets an entry message

**edit** Entry works fine if i subscribe to just one topic - so i guess subscribe_topics is problematic if it's a list ...
it's also not 100% reproducible - as a 2nd run did not trigger this behavior - which is odd - so maybe ignore this for now (i'm leaving it there should it reappear).

Reply:
This is very interesting, I will investigate further.

Reply:
don't spend too much time on it - it WAS interesting - but i couldn't reproduce it.
i also had 2 parallel clients connected at a time - but i was really surprised to see whitelist messages stop.